.devops/code-review-pipelines.yml

```diff
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-2938-tokenized-email
```

Comment:
To revert after merge of https://github.com/pagopa/pagopa-ecommerce-local/pull/167